hadoop-hdds/common/dev-support/findbugsExcludeFile.xml

```diff
@@ -27,6 +27,9 @@
     <Method name="update" />
     <Bug pattern="SF_SWITCH_FALLTHROUGH,SF_SWITCH_NO_DEFAULT" />
   </Match>
+  <Match>
+    <Class name="org.apache.hadoop.hdds.utils.db.CodecBuffer"></Class>
+  </Match>
```

Comment:
This can be removed now, 5cb7fae5b14fcd8169b63866b6407e6589dea1b1 fixed the warnings instead of suppressing them.